api/v1beta1/cinder_types.go

```diff
@@ -172,6 +186,32 @@ type Cinder struct {
 	Status CinderStatus `json:"status,omitempty"`
 }
 
+// DBPurge struct is used to model the parameters exposed to the Cinder cronJob
+type DBPurge struct {
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=30
+	// +kubebuilder:validation:Minimum=1
+	// Age is the DBPurgeAge parameter and indicates the number of days of purging DB records
+	Age int `json:"age"`
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default="1 0 * * *"
+	// Schedule defines the crontab format string to schedule the DBPurge cronJob
+	Schedule string `json:"schedule"`
+}
+
+// CinderDebug indicates whether certain stages of Cinder deployment should
+// pause in debug mode, or if debug is enabled when purging the DB.
+type CinderDebug struct {
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=false
+	// dbSync enable debug
+	DBSync bool `json:"dbSync"`
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=false
+	// DBPurge enable debug
+	DBPurge bool `json:"dbPurge"`
```

Comment:
Mention that this debug action just increases the verbosity of the command executed.

Reply:
Done